### DISCUSSION TITLE
Is there a way to authenticate users with oauth?

### DISCUSSION THREAD

**Discussion by Andrinn** [Glench/ExtPay#165] [Q&A]
My Chrome extension requires integration with the Salesforce API using OAuth. I'm wondering if it's feasible to leverage the user's extensionpay login credentials for OAuth authentication. If not, I might need to create an independent account system, which unfortunately would necessitate the user to log in twice – once for extensionpay and then again for my system.

**Comment by Glench** [comment]
It is not possible to use oauth, no. If you need something like that I would unfortunately have to recommend making your own system that is more suited to your needs.
